google-cloud-translate/acceptance/translate_v2_smoke_test.rb

```diff
@@ -51,7 +50,7 @@
 
     translation = translate.translate "How are you today?", to: "es", model: "base"
```

Comment:
Shouldn't this change to be "nmt". There are other tests below for nmt

Reply:
I left it as such because base model has to be updated to nmt if passed in, hence thought this could be a case to test that.